lib/core/services/authentication.service.ts

```diff
@@ -61,6 +61,16 @@ export class AuthenticationService {
         return this.alfrescoApi.getInstance().isLoggedIn();
     }
 
+    isLoggedInWith(provider: string) {
```

Comment:
missing return value type
```suggestion
    isLoggedInWith(provider: string): boolean {
```